docs/finalproject/roles.md

```diff
@@ -10,40 +10,34 @@ These roles should in total create a [balanced team](https://www.youtube.com/wat
 
 ## Volunteer Roles
 
-### Product Manager // Scrum Master
-
-**Time Commitment**: One day a week plus 2-4 hours during the week
-
-Scrum Masters are expected to coordinate the team and make sure everyone knows what they are working on. They can provide support in
+### Product Manager / Scrum Master
+Full role description is [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-manager)
```

Comment:
Links must make sense out of context

Reply:
Strange, I accepted your change to it, not sure why it didn't do it.

Reply:
updated.